includes/post-meta-box.php

```diff
@@ -131,6 +131,10 @@ function save_meta_box( $post_id ) {
 			update_post_meta( $post_id, 'podcast_filesize', $podcast_meta['filesize'] );
 			update_post_meta( $post_id, 'podcast_duration', $podcast_meta['duration'] );
 			update_post_meta( $post_id, 'podcast_mime', $podcast_meta['podcast_mime'] );
+
+			// Add enclosure meta data
+			$enclosure = $podcast_meta['url'] . PHP_EOL . $podcast_meta['filesize'] . PHP_EOL . $podcast_meta['podcast_mime'];
```

Comment:
In WordPress core [wp-includes/feed.php](https://github.com/WordPress/WordPress/blob/63ea8284a3e57ae289ce9ce39a4469cdc317906a/wp-includes/feed.php#L481) the hardcoded `"\n"` is used to explode the enclosure meta before it will be displayed, and so do you in [assets/js/edit.js:79](https://github.com/10up/simple-podcasting/pull/155/files#diff-1a9dc6a976d2043e93be6d50287f70f13ffb4008efe7d8cb8d491a2e6234703fR79) to combine partials into meta string.

Using `PHP_EOL` here might break the logic on non-Unix systems.